docs/sources/write/style-guide/style-conventions/index.md

```diff
@@ -142,7 +142,7 @@ To create a definition list, add the term on a new line then add a new line with
 
 For additional information, refer to the [Definition Lists](https://www.markdownguide.org/extended-syntax/#definition-lists) from the Markdown Guide.
 
-### Sorting list
```

Comment:
Should this be "Sort lists"?

Reply:
I bumped it to `Sorting lists` in this PR.... waffled on sorting vs sort. I was tripped up by the imperative here. I'll spin up a new PR to go with the simple verb `Sort` :-)